v2/go.sum

```diff
@@ -17,6 +17,7 @@ github.com/google/go-cmp v0.2.0/go.mod h1:oXzfMopK8JAjlY9xF4vHSVASa0yLyX7SntLO5a
 github.com/pmezard/go-difflib v1.0.0 h1:4DBwDE0NGyQoBHbLQYPwSUPoCMWR5BEzIk/f1lZbAQM=
 github.com/pmezard/go-difflib v1.0.0/go.mod h1:iKH77koFhYxTK1pcRnkKkqfTogsbg7gZNVY4sRDYZ/4=
 github.com/prometheus/client_model v0.0.0-20190812154241-14fe0d1b01d4/go.mod h1:xMI15A0UPsDsEKsMN9yxemIoYk6Tm2C1GtYGdfGttqA=
+github.com/spiffe/go-spiffe v1.0.0 h1:zP+DNibBYpALBw597lwikzhriUhMqSUU3zCjBAD+BFw=
```

Comment:
Hmm, where did this come from? What is pulling in go-spiffe v1?

Reply:
it's a
![image](https://user-images.githubusercontent.com/1189716/89681420-c5555d00-d8a9-11ea-8e8f-8e1e995b7ec5.png)

Reply:
not sure where this came from but it's cleaned up now